eng/source-build-patches/0024-Don-t-warn-on-NU1701-because-of-TFM-changes-in-sourc.patch

```diff
@@ -0,0 +1,54 @@
+Subject: [PATCH] Pin System.ComponentModel.Annotations to version that can be
+ added to sbrp
+
+dagood: (6.0) Note: this patch's filename doesn't match.
+---
+ .../src/Microsoft.Extensions.Options.DataAnnotations.csproj     | 2 +-
+ .../ref/Microsoft.Extensions.Options.csproj                     | 2 +-
+ .../src/Microsoft.Extensions.Options.csproj                     | 2 +-
+ 3 files changed, 3 insertions(+), 3 deletions(-)
+
+diff --git a/src/libraries/Microsoft.Extensions.Options.DataAnnotations/src/Microsoft.Extensions.Options.DataAnnotations.csproj b/src/libraries/Microsoft.Extensions.Options.DataAnnotations/src/Microsoft.Extensions.Options.DataAnnotations.csproj
+index bcf5ce56123..41bd9555c45 100644
+--- a/src/libraries/Microsoft.Extensions.Options.DataAnnotations/src/Microsoft.Extensions.Options.DataAnnotations.csproj
++++ b/src/libraries/Microsoft.Extensions.Options.DataAnnotations/src/Microsoft.Extensions.Options.DataAnnotations.csproj
+@@ -11,7 +11,7 @@
+   </ItemGroup>
+ 
+   <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETStandard'">
+-    <PackageReference Include="System.ComponentModel.Annotations" Version="$(SystemComponentModelAnnotationsVersion)" />
++    <PackageReference Include="System.ComponentModel.Annotations" Version="4.7.0" />
+   </ItemGroup>
+ 
+   <ItemGroup Condition="$(TargetFramework.StartsWith('net4'))">
+diff --git a/src/libraries/Microsoft.Extensions.Options/ref/Microsoft.Extensions.Options.csproj b/src/libraries/Microsoft.Extensions.Options/ref/Microsoft.Extensions.Options.csproj
+index c441c3ec38a..24c3e885eec 100644
+--- a/src/libraries/Microsoft.Extensions.Options/ref/Microsoft.Extensions.Options.csproj
++++ b/src/libraries/Microsoft.Extensions.Options/ref/Microsoft.Extensions.Options.csproj
+@@ -12,6 +12,6 @@
+     <ProjectReference Include="$(LibrariesProjectRoot)Microsoft.Extensions.Primitives\ref\Microsoft.Extensions.Primitives.csproj" />
+   </ItemGroup>
+   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+-    <PackageReference Include="System.ComponentModel.Annotations" Version="$(SystemComponentModelAnnotationsVersion)" />
++    <PackageReference Include="System.ComponentModel.Annotations" Version="4.7.0" />
+   </ItemGroup>
+ </Project>
+diff --git a/src/libraries/Microsoft.Extensions.Options/src/Microsoft.Extensions.Options.csproj b/src/libraries/Microsoft.Extensions.Options/src/Microsoft.Extensions.Options.csproj
+index ba7dcec688a..d3cfc425ac6 100644
+--- a/src/libraries/Microsoft.Extensions.Options/src/Microsoft.Extensions.Options.csproj
++++ b/src/libraries/Microsoft.Extensions.Options/src/Microsoft.Extensions.Options.csproj
+@@ -18,7 +18,7 @@
+   </ItemGroup>
+ 
+   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+-    <PackageReference Include="System.ComponentModel.Annotations" Version="$(SystemComponentModelAnnotationsVersion)" />
++    <PackageReference Include="System.ComponentModel.Annotations" Version="4.7.0" />
```

Comment:
This patch shouldn't exist at all. The current 5.0.0 version is last shipping version of that package and should be used. Isn't that already ingested in the source built package cache?

Reply:
This patch has been removed, compliments of you! Thanks